config/config.go

```diff
@@ -35,6 +35,10 @@ const (
 	// KeyS3DisableSSL is the configuration key that determines whether SSL is to be used.
 	// any value other than 'true' is treated as false.
 	KeyS3DisableSSL = "s3-disable-ssl"
+	// KeyS3DisableSSLVerify is the configuration key that determines whether to verify the
```

Comment:
Can you add tests for the new code here to https://github.com/kbase/blobstore/blob/master/config/config_test.go